addon/components/frost-info-bar.js

```diff
@@ -5,5 +5,6 @@ export default Component.extend({
   // == Properties ============================================================
 
   classNames: ['frost-info-bar'],
-  layout
+  layout,
+  hook: 'info-bar'
```

Comment:
@job13er at least in this case it _wasn't_ added and we caught it before a default was included, so no harm no foul